### PR TITLE
Converted data.js to data.jsonloading data from JSON instead of a JavaScript file 

### PR DESCRIPTION
**Pull request by AbhishekPSingh07**
### Problem:
As Stated in #17 The data was being loaded from data.js which is a javascript file. But the standard way to store data for webpages is to store it in a json file and load it into the webpage once it's used.

### Changes:
Added a file data.json which is the converted version of data.js to JSON format also changed some of the declaration of objects to remove redundancy in code. That is removed extra objects of mapping2018 ,mapping2019 and so on rather added the specified json object to the yeartotax object for better and easy utilisation.

So data.js was removed and data.json was added also the necessary import was changed in the App.svelte and now it imports from data.json
Screenshot 2023-04-10 at 8 40 07 AM

so initially the data.js file looked like:-
Screenshot 2023-04-10 at 8 37 02 AM
And Now after converting it into data.json it is as follows
Screenshot 2023-04-10 at 8 39 45 AM

### CODE REVIEW

**Comment by AbhishekPSingh07** [comment]
@Larry-Gan I did all the changes asked but was unable to create a PR without forking the repo.

File: src/lib/data.json [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+            "separate": 12000,
+            "together": 24000,
+            "head": 18000
+          }
+      },
+      "2019": {
+          "percents": [0.1, 0.12, 0.22, 0.24, 0.32, 0.35, 0.37],
+          "single": [0, 9700, 39475, 84200, 160725, 204100, 510300],
+          "seperate": [0, 9700, 39475, 84200, 160725, 204100, 306175],
+          "ss": 132900,
```
Comment (Larry-Gan) [inline src/lib/data.json:63]: It seems like you're missing entries for "together and "head" here. Is it possible to add them?

File: src/lib/data.json [hunk 2]
```diff
@@ -0,0 +1,219 @@
...
+            "budgetPercents": [0.1374, 0.1178, 0.1111, 0.0998, 0.1987, 0.0576, 0.0696, 0.0256, 0.0195, 0.0505, 0.0218, 0.0125, 0.0078, 0.0087, 0.0387, 0.0041, 0.0036, 0.0018, 0.0131],
+            "standardDeduction": {
+            "single": 12550,
+            "seperate": 12550,
+            "together": 25100,
+            "head": 18800
+        }},
+        "2022" : {
+            "percents": [0.1,0.12,0.22,0.24,0.32,0.35,0.37
+            ],
```
Comment (Larry-Gan) [inline src/lib/data.json:102]: Can you move this to the previous line?

File: src/lib/data.json [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+            "single": [0,10275,41775,89075,170050,215950,539900],
+            "seperate": [0,10275,41775,89075,170050,215950,332925],
+            "together": [0,20550,83550,178150,340100,431900,647850],
+            "head": [0,14650,55900,89050,170050,215950,539900],
+            "ss": 147000,
+            "budgetPercents": [0.1649,0.1528,0.1395,0.1305,0.104,0.0894,0.0513,0.033,0.0094,0.0254,0.0332,0.0111,0.0098,0.0106,0.0086,0.0049,0.0037,0.0019,0.016
+            ],
```
Comment (Larry-Gan) [inline src/lib/data.json:109]: Can you also move this to the previous line?

File: src/lib/data.json [hunk 4]
```diff
@@ -0,0 +1,219 @@
...
+            "head": [0, 14200, 54200, 86350, 164900, 209400, 523600],
+            "ss": 142800,
+            "budgetPercents": [0.1374, 0.1178, 0.1111, 0.0998, 0.1987, 0.0576, 0.0696, 0.0256, 0.0195, 0.0505, 0.0218, 0.0125, 0.0078, 0.0087, 0.0387, 0.0041, 0.0036, 0.0018, 0.0131],
+            "standardDeduction": {
+            "single": 12550,
+            "seperate": 12550,
+            "together": 25100,
+            "head": 18800
+        }},
+        "2022" : {
```
Comment (Larry-Gan) [inline src/lib/data.json:100]: For this entry I noticed that there weren't spaces between each of the entries on the list. Not sure how the copy paste went wrong, and this is a bit nitpicky, but is it possible for you to add them?

**Review by Larry-Gan — CHANGES_REQUESTED** [review]
Thank you so much for contributing! And I'm sorry about not being able to collaborate without forking, I added you as a contributor so you should be able to make branches now. Also, if it isn't too much of a hassle for you, can you make a branch for this change? It would make checking out your changes a bit easier on my side, but you don't have to do it since I kinda screwed up.

I went through the json in a bit greater detail and noticed some more things that could be changed. I'm sorry I missed them earlier, but can you respond to those suggestions too?

Also, I probably should've been clearer, but when I talked about screenshots, I meant the results of running `npm run dev`, not the code. Can you post a picture of the code working locally as well?
image

File: src/lib/data.json [hunk 5]
```diff
@@ -0,0 +1,219 @@
...
+      {
+        "value": "together",
+        "label": "Together"
+      },
+      {
+        "value": "head",
+        "label": "Head"
+      }
+    ],
+    "yearToTax": {
```
Comment (Larry-Gan) [inline src/lib/data.json:42]: Oh wait, one more thing, I just realized "yearToTax" is unecessary now as well. I think we originally had it to make it so our dictionary didn't get too big, but we really don't need it now that everything's in a json now, so can you remove "yearToTax" as well?

If you do this, you'll also need to remove yearToTax in lines 13 and 17 of App.svelte too.

**Review by Larry-Gan — CHANGES_REQUESTED** [review]
Sorry, I also missed "yearToTax" being redundant now too.

**Pull request closed without merging**